packages/web-app-files/tests/unit/components/SideBar/Actions/FileActions.spec.js

```diff
@@ -0,0 +1,408 @@
+import Vuex from 'vuex'
+import DesignSystem from 'owncloud-design-system'
+import stubs from '@/tests/unit/stubs/index.js'
+import { createLocalVue, mount } from '@vue/test-utils'
+import FileActions from '@files/src/components/SideBar/Actions/FileActions.vue'
+
+import GetTextPlugin from 'vue-gettext'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+localVue.use(Vuex)
+localVue.use(GetTextPlugin, {
+  translations: 'does-not-matter.json',
+  silent: true
+})
+
+const filesPersonalRoute = { name: 'files-personal' }
+
+const meta = {
```

Comment:
Could you look into extracting quite some of the mocks and fixtures into either https://github.com/owncloud/web/tree/master/__mocks__ and https://github.com/owncloud/web/tree/master/__fixtures__ ?

Background: We've introduced [actions in a contextmenu](https://github.com/owncloud/web/blob/master/packages/web-app-files/tests/unit/components/FilesList/ContextActions.spec.js) now and would like to make the underlying data reusable :)

Reply:
@pascalwengerter, i've refactored the fixtures into a seperate folder, please have a look